.goreleaser.yml

```diff
@@ -0,0 +1,48 @@
+# This is an example .goreleaser.yml file with some sane defaults.
```

Comment:
What is your idea moving this file into .github folder? next to `release.yml`?

Reply:
won't suggest it since by editing `${GOHOSTOS}` or using the command 
```bash
GOHOSTOS=$(go env GOHOSTOS) envsubst < ".goreleaser.yml" > .releaser.yml && release --rm-dist -f .releaser.yml
```
the can build it on their local machine besides its not github action config rather goreleaser app config file